smartanswers/config/deploy.rb

```diff
@@ -13,6 +13,8 @@
 set :rails_env, 'production'
 set :source_db_config_file, false
 
+set :rsync_options, "#{rsync_options} -v --delete-excluded --exclude 'test/artefacts/'"
```

Comment:
Ohh, good job figuring out `--delete-excluded`, assuming that does what it sounds like :)